pkg/gce-pd-csi-driver/utils.go

```diff
@@ -231,6 +231,13 @@ func containsZone(zones []string, zone string) bool {
 // (1) "context deadline exceeded", returns grpc DeadlineExceeded,
 // (2) "context canceled", returns grpc Canceled
 func CodeForError(err error) *codes.Code {
+	if err == nil {
```

Comment:
FYI: This [PR#1227](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1227/) moved CodeForError to the common pkg

Reply:
That PR isn't merged yet, so I think its ok to merge this as is.